t/shdict.t

```diff
@@ -399,7 +399,7 @@ GET /t
 value: 371
 err: nil
 --- error_log eval
-qr/\[TRACE\s+\d+ content_by_lua\(nginx\.conf:\d+\):7 loop\]/
+qr/\[TRACE\s+\d+ content_by_lua\(nginx\.conf:\d+\):7 loop]/
```

Comment:
Why this change? It's better to always escape `]` in regexes since it is a metachar per se though unpaired ones are usually harmless.